clientUtils/Util.ts

```diff
@@ -1141,3 +1141,23 @@ export function toRectangularMatrix<T, F>(arr: T[][], fill: F): (T | F)[][] {
         else return row
     })
 }
+
+// Wrapped because JSDOM does not support this method yet:
+// https://stackoverflow.com/questions/52968969/jest-url-createobjecturl-is-not-a-function/56643520#56643520
```

Comment:
It's probably nicer to fix this in a Jest setup script rather than here?
https://github.com/jsdom/jsdom/issues/1721#issuecomment-387279017

(Also I'm not entirely sure if we still need the hotfix, since createObjectURL is probably not called within the covered code path any more)

Reply:
You are right! I don't think we need this anymore, will get rid of it.